src/app/pages/ParatimeDashboardPage/LearningMaterials.tsx

```diff
@@ -99,6 +100,24 @@ const getContent = (t: TFunction) => {
         },
       },
       [Layer.cipher]: undefined,
+      [Layer.pontusx]: {
```

Comment:
```suggestion
      [Layer.pontusx]: undefined,
```

Reply:
Again, it's not that simple, because our current type definitions _demand_ content here.

We have a complicated auto-generated utility type here, which will need to be updated..

Reply:
Right, we don't even show Cipher, I was just looking at the code. Still... It would make sense to hide the whole learning section, if undefined, not to specify properties.

Reply:
I fixed the type definitions, so we can use undefined now.